package.json

```diff
@@ -213,16 +216,18 @@
     "valid-url": "^1.0.9",
     "vreme": "^3.0.2",
     "web3": "^0.20.7",
-    "web3-utils": "^1.2.4",
     "web3-eth-abi": "^1.2.6",
     "web3-stream-provider": "^4.0.0",
-    "xtend": "^4.0.1"
+    "web3-utils": "^1.2.4",
+    "xtend": "^4.0.1",
+    "@rsksmart/lumino-light-client-sdk": "file:../lumino-light-client-sdk"
```

Comment:
i think we should not commit this line since we are under development

Reply:
I think it doesn't heart anyone since we only need to change that later to a specific version, but if you want i can remove it. I mean it's just a help to the developer, if we remove it we need to specify somewhere that we need to link it, and npm link is really old at this point, file:.. syntax is better and more clear i think.

Reply:
Youre right!